pagopa/io-app
# feat: [IA-950] Customize the UI of the opt-in screen for reminders notifications

# ⚠️ This pr depends on #4132 ⚠️
Please review first #4132 in order to avoid duplicate review work.

## Short description

This PR updates the opt-in screen during the onboarding.

**_Please note_** that the switch to control **notification previews** is still **missing** here, and will be included in another PR.

| New User | Old User |
| - | - |
|![Simulator Screen Shot - iPhone 13 - 2022-10-13 at 13 35 28](https://user-images.githubusercontent.com/29163287/195589187-a4971ab6-3e85-4b55-bb1a-d4adfea70451.png) |![Simulator Screen Shot - iPhone 13 - 2022-10-13 at 13 35 02](https://user-images.githubusercontent.com/29163287/195589154-f3c5e009-ab4b-4713-b7a4-fd5ec5f2ef67.png) |




## List of changes proposed in this pull request
- Updated `OnboardingNotificationsPreferencesScreen`
- Updated `NotificationsPreferencesPreview`

## How to test
- Set `firstOnboarding` to `true` or `false` in the [dev server config](https://github.com/pagopa/io-dev-api-server/blob/02ea784cdf244759c9dd90cac42b60ad03491b58/src/config.ts#L63)
- Start dev server
- Perform a login
- You see the screen during onboarding
